tests/Filter/TagFilterTest.php

```diff
@@ -303,12 +303,13 @@ public function testTagFilterThatIsAllWhitespaceIsIgnored(): void
     private function expectDeprecationError(): void
     {
         set_error_handler(
-            static function ($errno, $errstr) {
+            static function (int $errNo, string $errStr, string $errFile, int $errLine) {
                 restore_error_handler();
-                throw new Exception($errstr, $errno);
+                throw new ErrorException($errStr, $errNo, filename: $errFile, line: $errLine);
```

Comment:
I changed it to `ErrorException` so that we do not capture unrelated exceptions by mistake (although the risk was already relatively low since we checked the message).

Additionally, I added the originating file/line parameters, which I think could be useful.